public/javascripts/moneyTracker.js

```diff
@@ -1,7 +1,11 @@
 var app = angular.module("App", ["xeditable"]);
 
+app.run(function() {
+        FastClick.attach(document.body);
+});
+
```

Comment:
This seems redundant, can you put this inside of the other app.run function? Or is there a reason why we need to run this in a separate instance?